SI/Sem.04/Pract.04/Task03.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+using namespace std;
+
+bool isPositive(int num)
+{
+	bool isPositive;
+	if (num >= 0)
+	{
+		isPositive = true;
+	}
+	else
+	{
+		isPositive = false;
+	}
+	return isPositive;
+}
+
+bool isNegative(int num)
+{
+	bool isNegative;
+	if (num < 0)
```

Comment:
num < 0 is a boolean expression, so it returns a boolean value that you can directly return